package.json

```diff
@@ -34,13 +36,22 @@
   },
   "browserslist": "electron 1.6",
   "lint-staged": {
-    "*.(js|jsx)": [
+    "*.*(js|jsx)": [
       "cross-env NODE_ENV=development eslint --cache --format=node_modules/eslint-formatter-pretty",
       "prettier --ignore-path .eslintignore --single-quote --write",
       "git add"
     ],
-    "*.(json|css|scss|md)|.(babelrc|prettierrc|eslintrc)": [
-      "prettier --ignore-path .eslintignore --write",
+    "*.json|.*(babelrc|eslintrc|stylelintrc)": [
+      "prettier --ignore-path .eslintignore --parser json --write",
+      "git add"
+    ],
+    "*.*(css|scss)": [
+      "stylelint --ignore-path .eslintignore --syntax scss --fix",
```

Comment:
How come we're using two css formatters here? Why not one? Does `stylelint` account for cases that `prettier` doesn't?

Reply:
`stylelint` caught a missing generic font error, which `prettier` couldn't.
`stylelint` seems to be the equivalent of `eslint` for styles, so I think it's better to keep it.